recaptcha_enterprise/snippets/src/main/java/recaptcha/accountdefender/AccountDefenderAssessment.java

```diff
@@ -60,12 +60,12 @@ public static void main(String[] args)
     String userIdentifier = "default" + UUID.randomUUID().toString().split("-")[0];
 
     // Change this to a secret not shared with Google.
-    final String HMAC_KEY = "SOME_INTERNAL_UNSHARED_KEY";
+    final String hmacKey = "SOME_INTERNAL_UNSHARED_KEY";
```

Comment:
It is a constant. The original [naming](go/java-style#s5.2.4-constant-names) was correct. Please, rollback.